jdi-light-angular-tests/src/test/java/io/github/epam/site/steps/States.java

```diff
@@ -0,0 +1,55 @@
+package io.github.epam.site.steps;
+
+import static com.epam.jdi.light.elements.composite.WebPage.openUrl;
+import static com.epam.jdi.light.settings.JDISettings.DRIVER;
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.StaticSite.loginForm;
+import static io.github.com.StaticSite.logout;
+import static io.github.com.StaticSite.userIcon;
+import static io.github.com.StaticSite.userName;
+
+import com.epam.jdi.light.elements.composite.WebPage;
+import io.github.com.StaticSite;
+import io.qameta.allure.Step;
+
+public class States {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'States' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)